data/sql/updates/pending_db_world/rev_1612452141389407700.sql

```diff
@@ -0,0 +1,142 @@
+-- Dire Maul start
+
+-- Decreased Guard Slip'kik's speed by 55%
+UPDATE creature_template SET `speed_walk`= 1.76 WHERE entry = 14323;
+
+-- Gordok's Spirits are not suppose to be attack-able / But they still should be in order to trigger the Smart_Scripts 
+UPDATE creature_template SET `unit_flags`= 131076 WHERE entry = 11446;
+
+-- Netherwalker is not suppose to grant XP when killed. As this is very abusable
+UPDATE creature_template SET `flags_extra`= 64 WHERE entry = 14389;
+
+-- Deleting mobs from pack 1 (they are not suppose to be there)
+DELETE FROM `creature` WHERE `guid` IN (247711);
+
+-- Inserting missing mob in pack 1 (this one is suppose to be here)
+DELETE FROM `creature` WHERE `guid`=247726;
+INSERT INTO `creature` (`guid`, `id`, `map`, `zoneId`, `areaId`, `spawnMask`, `phaseMask`, `modelid`, `equipment_id`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecs`, `wander_distance`, `currentwaypoint`, `curhealth`, `curmana`, `MovementType`, `npcflag`, `unit_flags`, `dynamicflags`, `ScriptName`) VALUES 
+(247726, 13036, 429, 0, 0, 1, 1, 0, 0, 293.428,  22.3743, -3.91569, 1.70123, 86400, 0, 0, 3998, 0, 0, 0, 0, 0, '');
```

Comment:
This GUID is already in-use of another NPC.

Change it.

Reply:
Are you sure you want to delete and override it

Reply:
Any GUID to recommend?

Reply:
You want to delete it in your first submission, now it's just recycling